pubnub-hyper/src/transport/hyper/mod.rs

```diff
@@ -47,7 +101,31 @@ impl Transport for Hyper {
         Ok(timetoken)
     }
 
-    async fn subscribe_request(&self, url: Uri) -> Result<(Vec<Message>, Timetoken), Self::Error> {
+    async fn subscribe_request(
+        &self,
+        request: request::Subscribe,
+    ) -> Result<(Vec<Message>, Timetoken), Self::Error> {
+        // TODO: add caching of repeating params to avoid reencoding.
+
+        // Prepare encoded channels and channel_groups.
+        let encoded_channels = EncodedChannelsList::from(request.channels);
+        // let encoded_channel_groups = EncodedChannelsList::from(req.channel_groups.into_iter());
```

Comment:
nitpick: no commented code

Reply:
I don't like this either. This particular instance is already removed in the next iteration.
For the future, I think I saw a clippy rule for this. I wonder why it doesn't work, maybe it needs to be enabled explicitly. I'll double-check.